DEPLOYMENT.md

```
@@ -0,0 +1,149 @@
+
+### Required for Production:
+```
+VITE_SPOTIFY_CLIENT_ID = bfdf4d48857a4b629a803511e2c4752e
+SPOTIFY_CLIENT_SECRET = [your_spotify_client_secret]
+NOCODE_API_KEY = [your_nocode_api_key]
+JWT_SECRET = [your_generated_jwt_secret]
+FRONTEND_URL = https://vibelines.vercel.app
+VITE_BACKEND_URL = https://vibelines.vercel.app
+```
+
+### Existing Variables (copy from your current .env):
```

markdownlint-cli2 (0.18.1)

Trailing punctuation in heading
Punctuation: ':'

(MD026, no-trailing-punctuation)